### PR TITLE
Tabs

### PR DESCRIPTION
**Pull request by 47souls**
Now tabs appear when clicking on map link at menu list and disaapear when clicking on other links. Also small changes where provided.

### CODE REVIEW

File: src/main/webapp/WEB-INF/views/main.jsp [hunk 1]
```diff
@@ -52,36 +51,38 @@
...
 								<li class="tab col m3 tabsBackgroundColor"><a class="white-text" href="#test3">Events</a></li>
 							</ul>
 						</div>
 					</div>
-				</div> 
+				</div>
+				<!-- Actual content of the page -->
 				<div class="content">
-					<div class="progress" ng-controller="markerCtrl" ng-show="progressBarVision">
+					 <div class="progress" ng-controller="markerCtrl" ng-show="progressBarVision">
```
Comment (Bmaks-github) [inline src/main/webapp/WEB-INF/views/main.jsp]: Strange spaces.

File: src/main/webapp/WEB-INF/views/main.jsp [hunk 2]
```diff
@@ -52,36 +51,38 @@
...
 						<div class="indeterminate"></div>
 					</div>
 					<ng-view></ng-view>
-				</div>
+				</div>
```
Comment (Bmaks-github) [inline src/main/webapp/WEB-INF/views/main.jsp]: Strange intend.

File: src/main/webapp/resources/app/css/menu-anim.css [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+#tabsRow.ng-hide {
+    height: 0;
+}
+.tabs {
+	height: 44px;
+	transition: 0.2s linear all;	
+}
+.tabs.ng-hide {
+    height: 0;
+}
```
Comment (Bmaks-github) [inline src/main/webapp/resources/app/css/menu-anim.css]: All styles in one css file "style.css".

File: src/main/webapp/WEB-INF/views/main.jsp (hunk 2 above)
Comment (RoxVer) [reply src/main/webapp/WEB-INF/views/main.jsp]: There is a great shortcut to edit indents (works for java, html etc.): Ctrl+Shift+F

File: src/main/webapp/resources/app/css/menu-anim.css (hunk 3 above)
Comment (47souls) [reply src/main/webapp/resources/app/css/menu-anim.css]: Thanks a lot) Just forgot to replace)

File: src/main/webapp/WEB-INF/views/main.jsp (hunk 2 above)
Comment (47souls) [reply src/main/webapp/WEB-INF/views/main.jsp]: Thank`s for advice)

File: src/main/webapp/WEB-INF/views/main.jsp (hunk 1 above)
Comment (47souls) [reply src/main/webapp/WEB-INF/views/main.jsp]: Replaced.

**Comment by ghost** [comment]
reviewed

**Pull request merged**